x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/normalizer/Normalizer.java

```diff
@@ -198,15 +198,15 @@ private double mergeRecursively(Iterator<NormalizerResult> scoresIter, Normaliza
      * @return true if the update is considered "big"
      */
     private static boolean isBigUpdate(double oldVal, double newVal) {
-        if (Math.abs(oldVal - newVal) >= 1.0) {
+        if (Math.abs(oldVal - newVal) >= 2.0) {
             return true;
         }
         if (oldVal > newVal) {
-            if (oldVal * 0.5 > newVal) {
+            if (oldVal * 0.33 > newVal) {
```

Comment:
This is a reduction though, right? Will cause more frequent renorms.

Reply:
It will cause fewer updates.  Previously a reduction from 1.0 to 0.4 would have been written to the index.  With this change it won't.

Reply:
Ah, it's multiplying the greater value. Missed that. :-)